users/romus/romus.h

```diff
@@ -0,0 +1,279 @@
+// This is a modified version of Let's Split keyboard from bbaserdem
+
+#ifndef USERSPACE
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?